packages/web-app-files/src/components/SideBar/TagsPanel.vue

```diff
@@ -85,6 +86,7 @@ export default defineComponent({
 
     const injectedResource = inject<Resource>('resource')
     const resource = computed<Resource>(() => unref(injectedResource))
+    const readonly = computed<boolean>(() => unref(resource).locked === true)
```

Comment:
```suggestion
    const readonly = computed(() => unref(resource).locked === true)
```

I'd prefer not stating the type explicitly, since it gets inferred properly.